### DISCUSSION TITLE
MacOS gamepad support?

### DISCUSSION THREAD

**Discussion by TIMISONG-dev** [LizardByte/community#139] [Sunshine] [answered]
How to enable gamepad support on MacOS?

**Comment by kotleni** [comment, marked as answer]
Gamepad support on macOS not implemented yet.

**Comment by kotleni** [comment]
I have pull request with gamepad implementation, you can test it.

https://github.com/LizardByte/Sunshine/pull/756

**Reply by ReenigneArcher** [reply]
@TIMISONG-dev please test and let us know if it works well or if you have any issues. I will try to merge in the next couple of days.
